_rules/non-visual-location-reference-825ule.md

```diff
@@ -0,0 +1,506 @@
+				</li>
+				<li>
+					<a href="https://www.w3.org/Consortium/sponsor/">Donate</a>
+				</li>
+				<li>
+					<a href="https://www.w3.org/Consortium/siteindex">Sitemap</a>
+				</li>
+			</ul>
+					
+		</div>
+	</div>
+</body>
+```
+
+#### Failed Example 2
+
+The content in the column on the right is part of the navigation. The user is told to find the navigation on the right (which is a visual indicator word) and the navigation has a navigation element, but there are 2 navigation elements on the page so the user doesn't know which one to use.
+
+```html
+<head>
+	<style>
+	.col-container {
+		display: table;
+		width: 100%;
+	}
+	.col {
+		display: table-cell;
+		padding: 16px;
+	}
+	</style>
+</head>
+<body>
+	<nav>
+		<ul>
+			<li>
+				<a href="https://www.w3.org/">W3C homepage</a>
+			</li>
+			<li>
+				<a href="https://www.w3.org/standards/">Standards</a>
+			</li>
+			<li>
+				<a href="https://www.w3.org/participate/">Participate</a>
+			</li>
+			<li>
+				<a href="https://www.w3.org/Consortium/membership">Membership</a>
+			</li>
+		</ul>
+	</nav>
+	<div class="col-container">
+		<div class="col">
+			<p>Find the navigation on the right, for the non-essential links</p>
+		</div>
+		<nav>
+			<div class="col">
+				<ul>
+					<li>
+						<a href="https://www.w3.org/Consortium/contact">Contact</a>
+					</li>
+					<li>
+						<a href="https://www.w3.org/Help/">Help and FAQ</a>
+					</li>
+					<li>
+						<a href="https://www.w3.org/Consortium/sponsor/">Donate</a>
+					</li>
+					<li>
+						<a href="https://www.w3.org/Consortium/siteindex">Sitemap</a>
+					</li>
+				</ul>
+			</div>
+		</nav>
+	</div>
+</body>
+```
+
+#### Failed Example 3
+
+The content in the column on the right is indicated with the word "box" (which is a shape indicator based on visual perception) and is also indicated by referencing the word "howdy", but the word "howdy" is in a different paragraph than the word "box".
+
+```html
+<head>
+	<style>
+	.col-container {
+		display: table;
+		width: 100%;
+	}
+	.col {
+		display: table-cell;
+		padding: 16px;
+	}
+	button {
+		background: #FAA;
+	}
+	</style>
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Click the box on the right, for a surprise</p>
+			<p>If you can't find the box here is a hint: it says "Howdy"!</p>
+		</div>
+		<div class="col">
+			<button onclick="alert('Surprise!')">Howdy</button>
+		</div>
+	</div>
+</body>
+```
+
+#### Failed Example 4
+
+This document is using the word "triangle" but in this case the triangle menu is on a different page.
+
+```html
+<body>
+	<p>On the <a href="/test-assets/SC1.3.3-triangle-menu-without-heading.html">information page</a> you can find more examples within the triangle menu</p>
+</body>
+```
+
+#### Failed Example 5
+
+This document is using the word "star" but in this case the star is in an iframe.
+
+```html
+<body>
+<p>More examples can be found when you look underneath the star or underneath the heading examples</p>
```

Comment:
"underneath the heading examples" is useless (and wrong) in that example.

Reply:
Is this fixed like this?

Reply:
Not really. The point is that the test asset used in that example **does not** have a heading. So "underneath the star" is enough. "underneath the heading example" is useless and is actually wrong because there is no such heading to look underneath.

Reply:
I added the heading.

The star cannot be recognized because it is not actually a star but an asterisk. So the heading is needed to make it a passed example. Do you agree?